db/queries/categories.js

```diff
@@ -0,0 +1,61 @@
+const db = require('../connection');
+/* Fetch Categories and count of all items in that particular*/
+
+const getCategory = (categoryId) => {
```

Comment:
We can rename this to `getCategories` since we are fetching multiple categories at once.